src/helpers.js

```diff
@@ -80,8 +80,27 @@ var insertTask = function(namespace, input, options) {
   });
 };
 
+var getModelData = function({query, limit, continuation, key, Model}, callback) {
```

Comment:
Bad name, no docs, also list function, then export statement following it..

Reply:
How about calling this `listTableEntries`?  And yes, this should have a comment before it describing what each of the parameters does.  And maybe call it "Table" instead of "Model"?

Reply:
Changed the method name to listTableEntries and the variable name to Table, along with adding docs.
https://github.com/taskcluster/taskcluster-index/pull/39/files#diff-b38f7254be1a9f817fd2e33c294481a1R97